draft-ietf-quic-applicability.md

```diff
@@ -709,6 +709,22 @@ new version uses the same process in reverse.  Servers disable validation of the
 old version, stop sending the old version in Version Negotiation packets, then
 the old version is no longer accepted.
 
+# Unreliable Datagram Service over QUIC
+
+{{?I-D.draft-ietf-quic-datagram}} specifies an QUIC extension to enable sending
+and receiving unreliable datagrams over QUIC. Other than with the use of UDP
+only, the datagram service in QUIC is congestion-controlled, and thereby
+supporting application designers to address the requirements in {{?RFC8085}}.
+QUIC datagrams are not flow-controlled and data chunks may get drop if the
+receiver is overloaded.
+
+The reliable transmission service of QUIC usually provides a stream-based
+interface to send and received data over eventually multiple QUIC streams.
+The datagram service, however, usually provides a message-based interface,
```

Comment:
```suggestion
The datagram service, however, provides a message-based interface,
```

Unless there's a reason to use usually and you're going to explain what happens when that's not the case, I think it's clearer without.

Reply:
Usually is there because you can also implement an message base interface for stream (see taps) and there is nothing in the protocol spec that says anything about the interface.